src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -67,6 +66,8 @@ internal enum WellKnownMember
 
         System_Reflection_Missing__Value,
 
+        System_Reflection_IntrospectionExtensions__GetTypeInfo,
+        System_Type__Module,
```

Comment:
I think it would be appropriate to keep System_Type__Module next to other System.Type members.